README.md

```diff
@@ -1,101 +1,106 @@
-   - Click the **Add a permission** button and then,
-   - Ensure that the **Microsoft APIs** tab is selected.
-   - In the *Commonly used Microsoft APIs* section, click on **Microsoft Graph**.
-   - In the **Delegated permissions** section, ensure that the right permissions are checked: **User.Read**, **Mail.Send** and **Files.ReadWrite**. Use the search box if necessary.
-   - Select the **Add permissions** button.
-
-<a name="configuring-sample-to-use-Azure-AD-tenant"></a>
-### Step 3:  Configure the sample to use your Azure AD tenant
-
-In the steps below, "ClientId" is the same as "Application ID" or "AppId".
-
-Open the solution in Visual Studio to configure the projects.
-
-#### Configure the client project
-
-1. In the *console-csharp-connect-sample* folder, rename the `appsettings.json.example` file to `appsettings.json`
-1. Open and edit the `appsettings.json` file to make the following change
-    1. Find the line where `ClientId` is set as `YOUR_CLIENT_ID_HERE` and replace the existing value with the application (client) ID of the `Console App for Microsoft Graph` application copied from the Azure portal.
-
-<a name="build-and-run-sample"></a>
-### Step 4: Build and run the sample 
-
-1. Open the sample solution in Visual Studio.
-2. Press F5 to build and run the sample. This will restore the NuGet package dependencies and open the console application.
-3. When prompted, authenticate with your Microsoft account and consent to the permissions that the application needs.
-4. Follow the prompts to send a message from your account to yourself or someone else.
-   
-## Questions and comments
-
-We'd love to get your feedback about the Microsoft Graph API Console App. You can send your questions and suggestions in the [Issues](https://github.com/microsoftgraph/console-csharp-connect-sample/issues) section of this repository.
-
-Questions about Microsoft Graph development in general should be posted to [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoftgraph). Make sure that your questions or comments are tagged with [microsoftgraph].
-
-## Contributing ##
-
-If you'd like to contribute to this sample, see [CONTRIBUTING.MD](/CONTRIBUTING.md).
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
-  
-## Additional resources
-
-- [Other Microsoft Graph Connect samples](https://github.com/MicrosoftGraph?utf8=%E2%9C%93&query=-Connect)
-- [Microsoft Graph](https://developer.microsoft.com/en-us/graph)
-
-## Copyright
-Copyright (c) 2019 Microsoft. All rights reserved.
+# Microsoft Graph C# Console Connect Sample
+
+## Table of contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)

```

Comment:
Make this line (and line 7 below) as clickable reference links to their respective sections.